client/login/wp-login/index.jsx

```diff
@@ -36,7 +36,7 @@ import RequestLoginEmailForm from '../magic-login/request-login-email-form';
 import { recordTracksEvent } from 'state/analytics/actions';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 
-class Login extends React.Component {
+export class Login extends React.Component {
```

Comment:
No side effects I can think of.
More complicated option is importing the default export and mocking `connect` & `localize` HoCs.